opensearchpy/client/indices.py

```diff
@@ -861,23 +864,23 @@ def get_field_mapping(
     )
     def put_alias(
         self,
-        index: Any,
-        name: Any,
         body: Any = None,
+        index: Any = None,
+        name: Any = None,
```

Comment:
FYI: This is a breaking change for anybody that were calling the method without naming the arguments.

Reply:
Looks like a miss :(

Care to open a bug, or if you have time fix the code generator?

Reply:
The number of users affected by this is probably limited, but perhaps it's worth calling it out in the release notes?

Reply:
I have made a bug for it here: https://github.com/opensearch-project/opensearch-py/issues/803.